addons/account/models/account_move.py

```diff
@@ -1179,6 +1215,9 @@ def date_key(move):
                 # In the form view, we need to compute a default sequence so that the user can edit
                 # it. We only check the first move as an approximation (enough for new in form view)
                 pass
+            elif move.quick_edit_mode:
+                # We suggest the next sequence as the default name of the new move
+                pass
```

Comment:
This means that you will always recompute the name when one of the dependencies changes